application/asset/js/asset-form.js

```diff
@@ -8,6 +8,7 @@
             Omeka.openSidebar(sidebar);
             Omeka.populateSidebarContent(sidebar, $(this).data('sidebar-content-url'));
             selectingForm = $(this).closest('.asset-form-element');
+            $(this).addClass('asset-selecting-button');
```

Comment:
Previously discussed, but look at separating this asset-block specific functionality into the asset block's own code rather than having it here in the generic asset form.